### PR TITLE
Ability to configure endpoints prefixes.

### PR DESCRIPTION
**Pull request by floor12**
Now its possible to pass to Client class constructor prefixes for basic endpoints and also for apple/google/samsung endpoints too. All your tests are passed. Now this lib could be used to work with Alfa Bank rest api (probably with some other banks too).

### CODE REVIEW

**Comment by floor12** [comment]
Hi! What about this pull request?

File: src/Client.php [hunk 1]
```diff
@@ -24,6 +23,10 @@ class Client
 
     const API_URI      = 'https://securepayments.sberbank.ru';
     const API_URI_TEST = 'https://3dsec.sberbank.ru';
+    const API_PREFIX_DEFAULT = '/payment/rest/';
+    const API_PREFIX_APPLE= '/payment/applepay/';
```
Comment (voronkovich) [inline src/Client.php]: Please, add whitespaces before equal signs

File: src/Client.php [hunk 2]
```diff
@@ -10,7 +10,6 @@
 use Voronkovich\SberbankAcquiring\Exception\ResponseParsingException;
 use Voronkovich\SberbankAcquiring\HttpClient\CurlClient;
 use Voronkovich\SberbankAcquiring\HttpClient\HttpClientInterface;
-use Voronkovich\SberbankAcquiring\OrderStatus;
```
Comment (voronkovich) [inline src/Client.php]: Why did you remove this line? The `OrderStatus` class is used here: https://github.com/voronkovich/sberbank-acquiring-client/blob/452244943e88284a25ee6172d05063d3732c6f9d/src/Client.php#L337

File: .gitignore [hunk 3]
```diff
@@ -4,3 +4,9 @@ composer.phar
 
 ### Phive
 /tools/
+
```
Comment (voronkovich) [inline .gitignore]: This changes are not needed. It's better to have such lines in the [global .gitignore](https://thoughtbot.com/blog/global-gitignore).

File: src/Client.php (hunk 2 above)
Comment (floor12) [reply src/Client.php]: Yes, it used here, but OrderStatus has the same namespace...   My linter always remove imports like this.
What for you  are asking to bring it back? Just for more readable code or something more?

Comment (voronkovich) [reply src/Client.php]: @floor12, you're right! I forgot that `OrdedStatus` has the same namespace. Sorry for confusion. Indeed such imports should be removed.

**Comment by floor12** [comment]
4 tests were added. Code was formatted. .gitignore commit was reverted. What you say now?
If you think its necessary, i can add tests with custom prefix for all endpoints, but i think 4 is enough.

**Comment by voronkovich** [comment]
@floor12, Thanks for the contribution!

**Comment by floor12** [comment]
Thank you for your lib. It helps me twice. Im glad to make it more flexible. 
Good luck.
